### PR TITLE
added logging to playground

### PR DESCRIPTION
**Pull request by akrasyuk**

### CODE REVIEW

File: ReactWindows/Playground.Net46/App.config [hunk 1]
```diff
@@ -3,4 +3,27 @@
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6"/>
     </startup>
+  <system.diagnostics>
+      <sources>
+        <source name="ReactWindows" switchValue="All">
```
Comment (akrasyuk) [inline ReactWindows/Playground.Net46/App.config]: Here we are relying on Trace Source from Tracer inside ReactNative.Net46

File: ReactWindows/Playground.Net46/App.config [hunk 2]
```diff
@@ -3,4 +3,27 @@
...
+        </source>
+      </sources>
+    <switches>
+      <add name="SourceSwitch" value="Warning" />
+    </switches>
+    <sharedListeners>
+      <add name="TextWriter" type="System.Diagnostics.TextWriterTraceListener" initializeData="ReactWindows.log" traceOutputOptions="ProcessId, DateTime, ThreadId">
+        <filter type="System.Diagnostics.EventTypeFilter" initializeData="All"/>
+      </add>
+      <add name="EventWriter" type="ReactNative.Net46.Tracing.CustomTraceListeners.ReactNativeLogTraceListener, ReactNative.Net46, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" initializeData="ReactWindowsEvent">
```
Comment (akrasyuk) [inline ReactWindows/Playground.Net46/App.config]: Our Custom trace listener usage

File: ReactWindows/ReactNative.Net46/ReactPage.cs [hunk 3]
```diff
@@ -43,7 +43,7 @@ protected ReactPage()
             });
         }
 
-        private ReactInstanceManager ReactInstanceManager => _reactInstanceManager.Value;
+        protected ReactInstanceManager ReactInstanceManager => _reactInstanceManager.Value;
```
Comment (akrasyuk) [inline ReactWindows/ReactNative.Net46/ReactPage.cs]: To allow us use this inside inherited classes.

ReactInstanceManager is used to get ReactContext. 

In blue we were using 
` var memberInfo = typeof(ReactPage).GetField("_reactInstanceManager", BindingFlags.NonPublic | BindingFlags.Instance);` to get that manager.

File: ReactWindows/Playground.Net46/App.xaml.cs [hunk 4]
```diff
@@ -119,5 +123,15 @@ private void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
         {
             throw new Exception("Failed to load Page...");
         }
+
+        #region Logging
+
+        private async Task ActivateLogging()
+        {
```
Comment (abodalevsky) [inline ReactWindows/Playground.Net46/App.xaml.cs]: probably we should store context in application class just for any case?

File: ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj [hunk 5]
```diff
@@ -62,7 +62,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\..\windows\packages\Newtonsoft.Json.12.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>..\packages\Newtonsoft.Json.12.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
```
Comment (abodalevsky) [inline ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj]: please be careful with path overwriting. here must be `$(SolutionDir)`

**Pull request merged**